norduser/service/fork.go

```diff
@@ -49,9 +50,13 @@ func handlePsError(out []byte, err error) error {
 func parseNorduserPIDs(psOutput string) []int {
 	pids := []int{}
 	for _, pidStr := range strings.Split(psOutput, "\n") {
-		pid, err := strconv.Atoi(strings.TrimSpace(pidStr))
+		pidStr = strings.TrimSpace(pidStr)
+		if pidStr == "" {
```

Comment:
There were many of these `failed to parse pid string:  strconv.Atoi: parsing "": invalid syntax`